ray-curator/ray_curator/stages/audio/metrics/get_wer.py

```diff
@@ -0,0 +1,58 @@
+from dataclasses import dataclass
+
+import editdistance
```

Comment:
New dependency? Could we use `nltk` or another existing dependency instead?

Reply:
it is in nemo_toolkit[asr]'s dependencies